### ISSUE TITLE
Send button not working in TB 91.0.1

### ISSUE THREAD

**Issue by Ulisus** [darktrojan/shrunked#44]
When i try to send an email with the new Thunderbird 91.0.1 i can push the Send button, but it has no function.

As soon as i disable Shrunked Image Resizer it works again.

**Comment by darktrojan** [comment]
Yeah, there's a bug in Thunderbird. I guess you have no dictionaries installed. You could [download and install one](https://addons.thunderbird.net/en-US/thunderbird/language-tools/) as a work-around.

**Comment by Ulisus** [comment]
That's it. After installing the dictionary it works like a charm. Thanks!

**Comment by McGerbick** [comment]
May I suggest to keep this issue open until the bug is fixed in Thunderbird 91? Then people coming by and looking at the issues will more likely find this one.

And/Or: Make a comment in the README.md or on the dedicated [web page](https://darktrojan.github.io/shrunked/).

**Comment by Ulisus** [comment]
You're right, i reopened it.

**Comment by McGerbick** [comment]
I've just updated Thunderbird to version 91.1.0 on one machine without installing any dictionary (as I actually wanted to). A short check showed that sending an e-mail now seems to work just fine.
